### PR TITLE
Rework confd templates to remove churn (requires confd change)

### PR DESCRIPTION
**Pull request by robbrockbank**
I've cut a confd release based on the two branches Cory implemented - install updated to pull in this release.

### CODE REVIEW

File: node_filesystem/templates/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+These templates write out a TOML file (bird.toml and bird6.toml) that is used 
+to tell confd which set of the main BIRD templates to use.
+
+Based off the node_mesh parameter, the TOML file generated either points to the
+full-mesh config files, or the no-mesh config files.
+
+Once confd writes out the appropriate TOML file, confd is restarted to pick up
+the change to the bird.toml and bird6.toml files.  Since there are two sets
+of changes, a change to the node_mesh parameter will result in a double
```
Comment (tomdee) [inline node_filesystem/templates/README.md]: I don't understand this double restart of confd thing? Won't it restart confd then confd will restart bird?

Ah! Or do you mean one restart for ipv4 and one for ipv6?

**Comment by tomdee** [comment]
LGTM

**Comment by lxpollitt** [comment]
Please holds off meeting this until I have also LGTM. I want to understand this since it sounds like HLD that I have not reviewed.

**Comment by tomdee** [comment]
@lxpollitt We chatted about this and I think @robbrockbank talked to you about it too.

The headline was
* This adds some complexity
* It's not strictly needed for scale testing (but would probably make lives easier)
* And has no effect/behavior change when using full mesh
* It makes scaling much better when using a RR.

I'm inclined to merge it, since we've gone to the effort of writing and testing it. WDYT?

**Comment by lxpollitt** [comment]
OK, so long as we are 100% convinced that confd failing halfway through updating some of its own templates can never result in us getting stuck in a broken state.

**Comment by tomdee** [comment]
@robbrockbank Can you comment on what Alex says above?

**Comment by robbrockbank** [comment]
@lxpollitt @tomdee 

confd writes temporary files which it moves into place, so there is no risk of a half written confd toml file.  Worst case is that the toml file is either not present (because it failed before writing the first one), or that the toml file is out of date.  However, in both cases this is rectified once confd is started again.

**Pull request merged**